crime-applications-adaptor/application/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/client/MaatCourtDataApiClient.java

```diff
@@ -21,4 +21,7 @@ EformStagingResponse retrieveOrInsertDummyUsnRecordInEformStaging(
 
   @GetExchange("/internal/v1/assessment/rep-orders/usn/{usn}")
   RepOrderState retrieveCrimeApplicationResultsByUsn(@PathVariable int usn);
+
+  @GetExchange("/api/internal/v1/assessment/rep-orders/rep-order-state/{repId}")
```

Comment:
Please remove /api as it is part of the base URL

Reply:
Oops! Good catch